app/controllers/catalog_controller.rb

```diff
@@ -1422,7 +1422,7 @@ def rearrange_groups_array
 
   def get_available_resources(kls)
     @edit[:new][:available_resources] = {}
-    kls.constantize.all.each do |r|
+    kls.constantize.where("type is null or type != 'ServiceTemplateAnsiblePlaybook'").each do |r|
```

Comment:
if there is an index on `type`, then Postgres can do an index scan. But since we're bringing back all that data, not sure what we buy.

I don't know another way to write this query.
good catch on the `type is null` because I tend to miss this null nuance with `type != 'abc'`.